src/literals.py

```diff
@@ -4,7 +4,7 @@
 
 """Collection of global literals for the charm."""
 
-OPENSEARCH_DASHBOARDS_SNAP_REVISION = "24"
+OPENSEARCH_DASHBOARDS_SNAP_REVISION = "1"
 
 SUBSTRATE = "vm"
 CHARM_KEY = "opensearch-dashboards"
```

Comment:
Is it ok to keep the `CHARM_KEY` as it is?

Reply:
Oops. Thanks